### PR TITLE
PB-73: Several UI improvement in import tool

### PR DESCRIPTION
**Pull request by ltshb**
- Placeholder translation updated
- removed the search bar for the geo catalogue, the regular search bar in the header should be use.
- Added color background on text search emphasize
- Added tooltip for truncated text in catalogue
- Truncate and added tooltip for long layer title in active layers

[Test link](https://sys-map.dev.bgdi.ch/preview/feat-pb-73-search-geo-catalogue-2/index.html)

### CODE REVIEW

File: tests/cypress/tests-component/TextTruncate.cy.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        cy.get('[data-cy="inner-element"]').realHover({ position: 'left' })
+        cy.get('.tippy-box').should('not.exist')
+        cy.get('[data-cy="inner-element"]').click('left')
+
+        cy.viewport(50, 480)
+        // Here unfortunately I could not find a way to correctly wait that the
+        // tippy correct got registered after the resize, therefore I've added a wait.
+        // I've tried spying on the initializeTippy method, but somehow the spy was never called.
+        // eslint-disable-next-line cypress/no-unnecessary-waiting
+        cy.wait(1000)
```
Comment (ltshb) [inline tests/cypress/tests-component/TextTruncate.cy.js:68]: @pakb I'm not sure if I want to keep this test, I don't like the 1s wait. Unfortunately I did not find a way to make this test reliable. Without this wait is fails time to time.
I tried to spy on `ResizeObserver` and/or `initializeTippy` but  without success the spy was never called. Maybe you have some better idea ?

Comment (ltshb) [reply tests/cypress/tests-component/TextTruncate.cy.js:68]: Here are some code that I tried
```javascript
// Assertions based on the spy

            const slotContent = 'My slot content'
            // const { wrapper, component } = cy.mount(TextTruncate, {
            //     slots: { default: slotContent },
            // })
            cy.mount(TextTruncate, {
                slots: { default: slotContent },
            }).as('mount')
            cy.get('@mount').then((wrapper) => {
                cy.spy(wrapper.component, 'initializeTippy').as('initializeTippy')

                cy.get('@initializeTippy').should('not.have.been.calledOnce')
                cy.get('[data-cy="inner-element"]').realHover({ position: 'left' })
                cy.get('[data-cy="outter-element"]').should('not.have.attr', 'aria-describedby')
                cy.get('.tippy-box').should('not.exist')
                cy.get('[data-cy="inner-element"]').click('left')

                cy.viewport(50, 480)
                cy.get('@resizeObserverSpy').should('have.been.calledOnce')
                cy.get('@initializeTippy').should('have.been.calledOnce')
                cy.get('[data-cy="inner-element"]').realHover({ position: 'left' })
                cy.get('[data-cy="outter-element"]').should('have.attr', 'aria-describedby')
                cy.get('.tippy-box').should('be.visible').contains(slotContent)
                cy.get('[data-cy="inner-element"]').click('left')

                cy.viewport(300, 480)
                cy.get('@initializeTippy').should('have.been.calledOnce')
                cy.get('[data-cy="inner-element"]').realHover({ position: 'left' })
                cy.get('[data-cy="outter-element"]').should('not.have.attr', 'aria-describedby')
                cy.get('.tippy-box').should('not.exist')
                cy.get('[data-cy="inner-element"]').click('left')
            })

            // Clean-up: restore the original ResizeObserver
            win.ResizeObserver = originalResizeObserver
```

**Review by pakb — APPROVED** [review]
as discussed, try this wait/intercept approach to replace the arbitrary 1000ms wait, and we are good to go

File: src/modules/menu/components/LayerCatalogue.vue [hunk 2]
```diff
@@ -14,16 +14,20 @@ const props = defineProps({
         type: Boolean,
         default: false,
     },
+    hasSearchBar: {
+        type: Boolean,
+        default: false,
+    },
```
[the remark this answers is not in the document]
Comment (ltshb) [reply src/modules/menu/components/LayerCatalogue.vue]: done

**Comment by ltshb** [comment]
Also the tests now seems to be quite stable on the CI, it is not the case locally using the cypress gui and headless ! I'll try another approach using events instead of fake url.

**Comment by ltshb** [comment]
I could not find anyway to make the tests non flaky, therefore I skipped the one that causing issues for the moment

**Pull request merged**